### ISSUE TITLE
Unicode error on isdaq01

### ISSUE THREAD

**Issue by dfujim** [dfujim/bfit#73]
When fitting with MIGRAD minimizer ( likely something to do with the minuit stdout output )

```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib64/python3.6/tkinter/__init__.py", line 1705, in __call__
    return self.func(*args)
  File "/home/bnmr/.local/lib/python3.6/site-packages/bfit/gui/tab_fit_files.py", line 709, in do_fit
    raise errmsg from None
  File "/home/bnmr/.local/lib/python3.6/site-packages/bfit/gui/tab_fit_files.py", line 705, in do_fit
    xlims=xlims)
  File "/home/bnmr/.local/lib/python3.6/site-packages/bfit/fitting/fitter.py", line 241, in __call__
    **kwargs)
  File "/home/bnmr/.local/lib/python3.6/site-packages/bfit/fitting/fitter_migrad_hesse.py", line 27, in _do_fit
    **kwargs)
  File "/home/bnmr/.local/lib/python3.6/site-packages/bfit/fitting/fit_bdata.py", line 231, in fit_bdata
    iter_obj.write(''.join(msg))
  File "/home/bnmr/.local/lib/python3.6/site-packages/tqdm/std.py", line 620, in write
    fp.write(s)

UnicodeEncodeError: 'ascii' codec can't encode characters in position 25-99: ordinal not in range(128)
```

**Comment by dfujim** [comment]
Confirmed. The fix is this: 

```python
try: 
   s = str(m.fmin)
except UnicodeEncodeError:
  s = repr(m.fmin)
print(s)
```

**Comment by dfujim** [comment]
Fix added in 946c7b1836c0aea7a30dae908597a1cb36163d81
